.github/workflows/build-release.yml

```diff
@@ -48,7 +50,7 @@ jobs:
       run: |
         pip3 install meson==0.55.3 ninja
         brew install advancecomp automake brotli nasm pkg-config
-      if: matrix.platform == 'darwin-x64'
+      if: matrix.platform == 'darwin-x64' || matrix.platform == 'darwin-arm64v8'
```

Comment:
Might be able to use something like `if: contains(matrix.platform, 'darwin')` here.

Reply:
Ah, thanks. I'm quite unfamiliar with the github workflow syntax!